src/KafkaFlow.Abstractions/Configuration/IConsumerConfigurationBuilder.cs

```diff
@@ -136,19 +136,19 @@ public interface IConsumerConfigurationBuilder
         /// <summary>
         /// Sets the strategy to choose a worker when a message arrives
         /// </summary>
-        /// <typeparam name="T">A class that implements the <see cref="IDistributionStrategy"/> interface</typeparam>
+        /// <typeparam name="T">A class that implements the <see cref="IWorkerDistributionStrategy"/> interface</typeparam>
         /// <param name="factory">A factory to create the instance</param>
         /// <returns></returns>
-        IConsumerConfigurationBuilder WithWorkDistributionStrategy<T>(Factory<T> factory)
-            where T : class, IDistributionStrategy;
+        IConsumerConfigurationBuilder WithWorkerDistributionStrategy<T>(Factory<T> factory)
+            where T : class, IWorkerDistributionStrategy;
 
         /// <summary>
         /// Sets the strategy to choose a worker when a message arrives
         /// </summary>
-        /// <typeparam name="T">A class that implements the <see cref="IDistributionStrategy"/> interface</typeparam>
+        /// <typeparam name="T">A class that implements the <see cref="IWorkerDistributionStrategy"/> interface</typeparam>
         /// <returns></returns>
         IConsumerConfigurationBuilder WithWorkDistributionStrategy<T>()
```

Comment:
The name should be consistent with the other method.
```suggestion
        IConsumerConfigurationBuilder WithWorkerDistributionStrategy<T>()
```

Reply:
WithWorkDistributionStrategy was renamed to WithWorkerDistributionStrategy as suggested